src/models/ErrorResponse.js

```diff
@@ -1,9 +1,8 @@
 // @flow
```

Comment:
If this files does not correspond to an entity in our database I would suggest to move it out of models.